### ISSUE TITLE
Combobox auto fill drop down doesn't read to JAWS 2018/Chrome 65

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/aria-practices#670]
It doesn't work in Canary either
Version 68.0.3410.0 (Official Build) canary (64-bit)
https://www.w3.org/TR/wai-aria-practices-1.1/examples/combobox/aria1.1pattern/grid-combo.html

Same for this Combox examples not working in Chrome 65 and JAWS 2018

https://www.w3.org/TR/wai-aria-practices-1.1/examples/combobox/aria1.1pattern/listbox-combo.html

However, this example works with the same version of JAWS and Chrome 65
https://dequelabs.github.io/combobo/demo/

**Comment by DavidMacDonald** [comment]
Just noticed the 2018 update to Jaws
http://www.freedomscientific.com/Downloads/jaws/JAWSWhatsNew#Enhancements

> "addressed several issues reported by Cerner Corporation including: JAWS not reading list boxes correctly in some situations"

**Comment by mcking65** [comment]
@DavidMacDonald, do you see a possible bug in the example? I'm interpreting this issue as a potential JAWS or browser bug. Assuming a JAWS or browser bug, I'm closing as won't fix and not for practices. If I'm mistaken, please re-open and explain.
